extensions/storage/README.md

```diff
@@ -0,0 +1,78 @@
+# Object Storage Extension Specification
+
+- **Title: Object Storage**
+- **Identifier: storage**
+- **Field Name Prefix: -**
+- **Scope: Item, Assets**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+- **Owner**: @davidraleigh
+
+This document explains the fields of the Object Storage Extension to a STAC Item and an Asset. This does not cover NFS solutions provided by PaaS cloud companies. Allows user to specify details related to cloud storage access and costs for assets and their STAC item.
+
+- [Example (Landsat 8)](examples/example-landsat8.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Asset fields
+
+| Field Name  | Type   | Description |
+| ----------- | ------ | ----------- |
+| storage:cloud_platform        | string    | The [cloud provider](#providers) where data is stored |
```

Comment:
```suggestion
| storage:platform        | string    | The [cloud provider](#providers) where data is stored |
```

maybe just platform?

Reply:
Yes, shorter is better and cloud seems superfluous

Reply:
I was avoiding platform because C++ and enums and platform clashing with stac item platform. But I can make it `platform` for this pull request and make it `storage_platform` in protobuf.

Reply:
https://github.com/radiantearth/stac-spec/pull/991/commits/09510ce3e91b73010a93991defdd51856c47c3b1